OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            ምስለ፡ ገብራ፡
+                                        <persName ref="PRS12911KidanaK"><hi rend="rubric">ኪዳነ፡ 
+                                            ክርስቶስ፡</hi></persName> ለዓለመ፡
+                                        ዓለም፡ አሜን<hi rend="rubric">፨</hi>
+                                    </explicit>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i3.2.19.2">
+                                    <locus target="#60va"/>
+                                    <title type="complete" ref="LIT4274AkkonuB#Stanza18"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i3.2.20">
+                                <locus from="60va" to="61va"/>
+                                <title type="complete" ref="LIT5673AkkonuColl#AkkColl19"/>
+                                <textLang mainLang="gez"/>
+                                
+                                <msItem xml:id="ms_i3.2.20.1">
+                                    <locus from="60va" to="61rb"/>
+                                    <title type="complete" ref="LIT3605Miracle"/>
+                                    <textLang mainLang="gez"/>
+                                    
+                                    <incipit type="inscriptio" xml:lang="gez">
+                                        <locus target="#60va"/>
+                                        <hi rend="rubric">ተአምሪሃ፡ ለእግዝእትነ፡ ቅድስት፡ ድንግል፡ ማርያም፡</hi>
+                                        ወላዲተ፡ አምላክ፡
+                                    </incipit>
+                                    
+                                    <incipit type="supplication" xml:lang="gez">
+                                        <locus from="60va" to="60vb"/>
+                                        ጸሎታ፡ ወበረከታ፡ ወምሕረተ፡ ፍቁር፡ ወልዳ፡ የሃሉ፡ <cb n="b"/>ምስለ፡ 
+                                        ገብራ<hi rend="rubric">፨</hi> 
+                                        <persName ref="PRS12911KidanaK"><hi rend="rubric">ኪዳነ፡ ክርስቶስ፡</hi></persName>
+                                        ለዓለመ፡ ዓለም፡ አሜን<hi rend="rubric">፨</hi>
+                                    </incipit>
+                                    
+                                        <incipit xml:lang="gez">
+                                            <locus target="#60vb"/> ወሀሎ፡ <hi rend="rubric">፩</hi>
+                                            ብእሲ፡ መስተጋድል፡ ፈድፋደ፡ በከዊነ፡ ስምዕ፡ በእንተ፡ ስመ፡ ክርስቶስ፡ ዘስሙ፡
+                                            ጊዮርጊስ፡ ሐዲስ<hi rend="rubric">፨</hi> ወአሐተ፡ ዕለተ፡ እንዘ፡ ሀሎ፡
+                                            ተጽዒሮ፡ እምብዝኃ፡ ኵነኔ፡ ኖመ፡ ወበ<hi rend="rubric">፲</hi> ወ<hi rend="rubric">፩</hi>
```

Comment:
I think we might discuss in a future meeting how to consistently encode the sequence numeral+word (or numeral+ወ+numeral+word) in those cases where no colon ፡ is written between them.

Reply:
Yes!